### ISSUE TITLE
Patchouli integration might be broken?

### ISSUE THREAD

**Issue by gardenappl** [dphaldes/Tome-of-Blood#2]
I get this error in the log while loading:
```
[10Sep2023 01:06:06.099] [Worker-ResourceReload-0/ERROR] [patchouli/]: Failed to load book tomeofblood:example defined by mod tomeofblood, skipping
java.lang.IllegalArgumentException: Book tomeofblood:example has use_resource_pack set to false. This behaviour was removed in 1.20. The book author should enable this flag and move all book contents clientside to /assets/, leaving the book.json in /data/. See https://vazkiimods.github.io/Patchouli/docs/upgrading/upgrade-guide-120 for details.
        at vazkii.patchouli.common.book.Book.<init>(Book.java:145) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at vazkii.patchouli.common.book.BookRegistry.loadBook(BookRegistry.java:90) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at vazkii.patchouli.common.book.BookRegistry.lambda$init$3(BookRegistry.java:75) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
        at vazkii.patchouli.common.book.BookRegistry.init(BookRegistry.java:70) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at vazkii.patchouli.forge.common.ForgeModInitializer.onInitialize(ForgeModInitializer.java:76) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at vazkii.patchouli.forge.common.__ForgeModInitializer_onInitialize_FMLCommonSetupEvent.invoke(.dynamic) ~[Patchouli-1.20.1-81-FORGE.jar%23689!/:1.20.1-81-FORGE]
        at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.5.jar%23127!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.5.jar%23127!/:?]
        at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.5.jar%23127!/:?]
        at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:107) ~[javafmllanguage-1.20.1-47.1.3.jar%23796!/:?]
        at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:124) ~[fmlcore-1.20.1-47.1.3.jar%23795!/:?]
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Comment by dphaldes** [comment]
Thanks for letting me know! I don't do anything with Patchouli right now, but I will need to migrate the format nonetheless!

**Comment by gardenappl** [comment]
> I don't do anything with Patchouli right now

Oh, I didn't realize, lol. It's just that this exact issue is also affecting a whole bunch of other mods on 1.20 (that do actually add Patchouli docs) so I kinda just reported them all.

**Comment by dphaldes** [comment]
Yeah. Patchouli changed format in 1.20 https://vazkiimods.github.io/Patchouli/docs/upgrading/upgrade-guide-120/

**Comment by dphaldes** [comment]
This was fixed in v0.3.0 :)
